task_name: pr2_tasks/DMPJointSpaceWaypointTask
dmp_controller_name: r_arm_dmp_joint_position_controller
control_cost_weight: 0.0
waypoint_cost_weight: 100000.0
goal_cost_weight: 100.0
num_time_steps: 100
num_dimensions: 7
movement_duration: 2.0
execution_duration: 3.0
dmp_sampling_frequency: 1000.0
root_frame: torso_lift_link
tip_frame: r_gripper_tool_frame
run_offline: true

task_dmp_id: 5
reset_dmp_id: 6

first_trial: 1
last_trial: 200

reset_movement_duration: 5

joint_start: [-1.0, 0.0, 0.0239, -0.004, -0.392, -0.646, -0.001]
joint_goal: [0.0, 0.0, 0.0239, -0.004, -0.392, -0.646, -0.001]

cart_waypoint: [0.7, -0.5, 0.2]
cart_waypoint_time: 1.0

num_rollouts: 8
num_reused_rollouts: 4
#noise_stddev: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
#noise_stddev: [10.0, 10.0, 10.0, 10.0, 10.0, 10.0, 10.0]
noise_stddev: [1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0]
noise_decay: [0.99, 0.99, 0.99, 0.99, 0.99, 0.99, 0.99]

dmp:
  package_name: pr2_tasks
  library_base_directory_name: library
  library_directory_name: dmp_test_lib
  data_directory_name: dmp_test_data
  debug_directory_name: dmp_test_debug
  execution_duration: 2.0
  can_sys_cutoff: 0.001
  k_gain_icra2009: 50.0
  alpha_z_nips2003: 10.0

lwr:
  rfs_width_boundary: 0.7
  use_offset: no
  num_rfs: 20
